database/migration_runner_env.go

```diff
@@ -0,0 +1,87 @@
+package database
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+// Constants representing environment variable keys for migration configuration.
+const (
+	envKeyPrefix       = "MIGRATION"
```

Comment:
I think it's better to have duplicated prefixes in the constants below ("MIGRATION_DSN", "MIGRATION_OPERATION"), rather than having a method to concatenate them. It will be much more readable, and easier to search where an env var is used, i.e. when you see an env definition in a deployment config, and would like to know in which package it's used. If env vars defined as a full name, you can just search across all repos and find it.